### PR TITLE
modify sanitytests/fpga, can sythesize successfully in vivado

### PR DESCRIPTION
**Pull request by JACKLIAO0**
modified file sanitytests/fpga/src/DesignKeyWrapper.scala and sanitytests/fpga/src/FpgaHarness.scala, to make sythesize with 50MHz(max 80MHz) clock frequency in vivado can work correctly.

### CODE REVIEW

**Review by sequencer — CHANGES_REQUESTED** [review]

**Comment by sequencer** [comment]
please split arty100t from fpga

**Comment by JACKLIAO0** [comment]
I have added a case in Top config to modify the FPGAFrequencyKey.

**Comment by JACKLIAO0** [comment]
> please split arty100t from fpga

add a new branch named arty100T in playground?

**Comment by sequencer** [comment]
Provide both arty and vcu at the same time

File: sanitytests/fpga/src/FPGATest.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ object FPGATest extends TestSuite {
 //      val outputDirectory = os.pwd / "out" / "VCU118"
 //      os.remove.all(outputDirectory)
 //      os.makeDir(outputDirectory)
-//      val configs = Seq(classOf[FPGATestConfig], classOf[freechips.rocketchip.system.DefaultConfig])
```
Comment (sequencer) [inline sanitytests/fpga/src/FPGATest.scala:23]: Change FPGATest to VCU118Test and Arty100TTest

**Review by sequencer — CHANGES_REQUESTED** [review]

**Comment by JACKLIAO0** [comment]
@sequencer there is an error that i have no idea how to reslove, please have a see

**Pull request closed without merging**